### ISSUE TITLE
[feat] Carbon Neutral Mining - heat reusage / decentralized aspects

### ISSUE THREAD

**Issue by fmhc** [dogecoin/dogecoin#2120]
Hello there,

TL;DR: much wow - use energy twice - make carbon footprint zero with decentralized mining hardware, provided by (elon funded? community funded?) open hardware company which we need to open up now, can be bought with dogecoin. watercooled mining gear heats up your home, probably very much smart energy connected with solar/battery/additional computing power like your tesla's AI chips and other plugins by elon army. important thing: according to laws of thermodynamics, all (lets say 99.9%, roast me later while AMA me empty) the energy you put into a computing device is transformed into heat. There have been studies (e.g. Microsoft 2011 https://www.microsoft.com/en-us/research/publication/the-data-furnace-heating-up-with-cloud-computing/) about the potential of using micro cloud datacenters (1 rack sized) for heating up your house. I once had a server in my moms garage and then i made a startup out of that thing being loud and heating up my small homeoffice room. I started a company in 2012 and produced a prototype of a 1-rack-system with watercooling heating up a house in hamburg in 2015, we got EU-funding but didn't make it pivoting to datacenter consultant stuff or other ideas we had at that time - we would have needed a 1.5m grant to make it sustainable, but did not wanted to accept money by e.g. a consortium backed by ABB because they do nuclear shit and we wanted to stay true to real green values - whatever, get my idea of using energy twice with simple watercooling / heat distribution & production where it's needed, please :D

The basic idea from that was to build up a decentralized micro datacenter architecture where the security comes from decentralization (you can put blockchain apps / network instead of cloud services here) which has certified green power input and is double-using the energy we put in to heat that building its integrated in. Open Hardware Aspects and a modular design would make it possible to have different operating models and ensure that everyone could participate in green heating (a datacenter provider saves money from not having cooling costs; a home owner gets heat for free). You can check out the idea here: https://vimeo.com/50919936 (german voiced startup video from 2012). 

I would like to give everything I have (drafts of the system, any documentation and lots of time for AMAs on that or project support) to a community who is interested in going this path of carbon neutral, decentralized hardware support for (preferrably) dogecoin or any L2 approach to make dogecoin smart and more efficient. This was a startup project of mine, I would like to refactor that with anyone who is interested in going a sharing community / open hardware approach.

Addon to this idea: I would like to introduce a Dogecoin sidechain with that, based temporary on any other valid nice L1 chain, to keep track of co2 consumption and energy certificates for dogecoin mining. this concept could even be adopted by bitcoin to keep track of mining and have green bitcoin (and green dogecoin). send your thoughts please.

**Comment by patricklodder** [comment]
Hi! Great idea! Thank you!

I think a better place to discuss this would be https://www.reddit.com/r/dogecoindev as this doesn't necessarily need any changes to Dogecoin Core. I'm closing this here.

**Comment by fmhc** [comment]
okay thank you for guiding me here :) I've opened a topic on reddit for anyone who would like to follow up on this. https://www.reddit.com/r/dogecoindev/comments/nkverl/carbon_neutral_dogecoin_mining_hardware_ideas/

**Comment by fmhc** [comment]
How can I avoid auto-removal of my new topic from reddit with a fresh account? I have no Karma, cant post links, even when i remove the links the article gets removed and I dont know why. would be great if you could help me in reddit or tell me which mod i can ask for that, thanks a lot for your time!
